src/prob/tfopf.jl

```diff
@@ -0,0 +1,55 @@
+export run_tfopf, run_ac_tfopf, run_dc_tfopf
+
+""
+function run_ac_tfopf(file, solver; kwargs...)
+    return run_tfopf(file, ACPPowerModel, solver; kwargs...)
+end
+
+""
+function run_dc_tfopf(file, solver; kwargs...)
+    return run_tfopf(file, DCPPowerModel, solver; kwargs...)
+end
+
+""
+function run_tfopf(file, model_constructor, solver; kwargs...)
+    return run_generic_model(file, model_constructor, solver, post_tfopf; kwargs...)
+end
+
+""
+function post_tfopf(pm::GenericPowerModel)
+    variable_voltage(pm)
+    variable_generation(pm)
+    variable_line_flow(pm)
+    variable_dcline_flow(pm)
+    variable_phase_shift(pm)
+    variable_voltage_tap(pm)
```

Comment:
The current convention in power models is that variables and constraints are based on complex numbers.  So in this case, the variable and constraint signature should be along the line of "variable_transformation" or "variable_oltp".  Then inside "variable_transformation" you call "variable_voltage_shift" and "variable_voltage_tap".  For example see, https://github.com/lanl-ansi/PowerModels.jl/blob/master/src/core/variable.jl#L182